src/Http/Http.Results/test/ResultsTests.cs

```diff
@@ -737,7 +737,7 @@ public void Json_WithAllArgs_ResultHasCorrectValues()
         var options = new JsonSerializerOptions();
         var contentType = "application/custom+json";
         var statusCode = StatusCodes.Status208AlreadyReported;
-            
+
```

Comment:
Can this be reverted to not clutter source control history? Maybe the next time this file needs to be edited, it can be formatted.